zeppelin-server/src/test/java/org/apache/zeppelin/rest/AbstractTestRestApi.java

```diff
@@ -322,6 +347,13 @@ protected void describeMismatchSafely(String item, Description description) {
     };
   }
 
+  //Create new Setting and return Setting ID
+  protected String createTempSetting(String tempName) throws IOException {
+
+    InterpreterGroup interpreterGroup =  ZeppelinServer.notebook.getInterpreterFactory().add(tempName,"newGroup",new InterpreterOption(false),new Properties());
```

Comment:
This line seems to be longer then 100 chars, which is discouraged by [Styleguide](https://github.com/apache/incubator-zeppelin/blob/master/CONTRIBUTING.md#code-convention)